run.sh

```diff
@@ -1,20 +1,64 @@
 #!/bin/bash
 
-access_id=${SUMO_ACCESS_ID:=$1}
-access_key=${SUMO_ACCESS_KEY:=$2}
-receiver_url=${SUMO_RECEIVER_URL:=https://collectors.sumologic.com}
-collector_name=${SUMO_COLLECTOR_NAME:=collector_container}
-sources_json=${SUMO_SOURCES_JSON:=/etc/sumo-sources.json}
+SUMO_ACCESS_ID=${SUMO_ACCESS_ID:=$1}
+SUMO_ACCESS_KEY=${SUMO_ACCESS_KEY:=$2}
+SUMO_RECEIVER_URL=${SUMO_RECEIVER_URL:=https://collectors.sumologic.com}
+SUMO_COLLECTOR_NAME=${SUMO_COLLECTOR_NAME_PREFIX:=''}${SUMO_COLLECTOR_NAME:=`cat /etc/hostname`}
```

Comment:
when the collector starts up, if a collector name is not provided, we use the name from
```
java.net.InetAddress.getLocalHost.getHostName
```
which should return the same value on Linux.  So I think this step is not necessary?